### PR TITLE
Update deprecated commands

### PR DESCRIPTION
**Pull request by nak3**

### CODE REVIEW

**Comment by sdodson** [comment]
@nak3 i haven't kept up with things well enough, do you know in which version the new commands were introduced?

**Comment by nak3** [comment]
@sdodson It was introduced by these report and PR, and included in OSEv3.0.1.0.

https://github.com/openshift/origin/issues/2461
https://github.com/openshift/origin/pull/3557

At the OSEv3.0.1.0, we can still run the command without `ca`, but it outputs warning message.

~~~
# oadm version
oadm v3.0.1.0-1-5-ge51f583-dirty
kubernetes v1.0.0

# oadm create-server-cert
Command "create-server-cert" is deprecated, Use 'oadm ca' instead.
error: at least one hostname must be provided
see 'oadm create-server-cert -h' for help.
~~~

**Comment by jaryn** [comment]
Can one of the admins verify this patch?

**Comment by abutcher** [comment]
@sdodson Would we need to wrap these in a conditional to continue to support 3.0 ?

**Comment by sdodson** [comment]
> @sdodson Would we need to wrap these in a conditional to continue to support 3.0 ?

Yeah, I'd rather just make the conditional 3.0 versus 3.1 to keep it simple. The old commands continue to work just fine through 3.1.

**Comment by jaryn** [comment]
Can one of the admins verify this patch?

**Comment by sdodson** [comment]
@jaryn Are you going to bump each PR daily? I don't think that's necessary.

**Comment by detiber** [comment]
:+1: on making it conditional on 3.1 (and the origin equivalent)

**Comment by arilivigni** [comment]
Testing job has been triggered.

**Comment by arilivigni** [comment]
Internal Jenkins error during triggering.
 No test results found.

**Comment by detiber** [comment]
@nak3 can you update this to resolve the merge conflict and to make the use of the new command optional based on the openshift.common.version_greater_than_3_1_or_1_1 fact?

**Comment by nak3** [comment]
@detiber Sure, I updated.

File: roles/openshift_node_certificates/tasks/main.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@
 
 - name: Generate the node server certificate
   command: >
-    {{ openshift.common.admin_binary }} create-server-cert
+    {{ openshift.common.admin_binary }} {{ 'ca create-master-certs' if (openshift.common.version_greater_than_3_1_or_1_1 | bool) else 'create-master-certs' }}
```
Comment (detiber) [inline roles/openshift_node_certificates/tasks/main.yml]: Should be create-server-certs instead of create-master-certs here.

**Comment by nak3** [comment]
Fixed. Thank you @detiber It was a big mistake.

**Comment by detiber** [comment]
:+1: @brenton

**Comment by tbielawa** [comment]
This PR has been inactive for quite some time. Please update, fix any conflicts, and reopen this PR if this is still a priority you would like to see action on.

**Pull request closed without merging**